vagrant/provisioning/arkcase-all.yml

```diff
@@ -114,10 +114,10 @@
       when: enable_microservices | default(false)
     - role: arkcase-ldap-authority
       tags: [core, upgrade, arkcase-ldap-authority, arkcase-microservices, marketplace]
-      when: enable_microservices | default(false) and enable_saml_microservice | default(false)
+      when: enable_microservices | default(false) and not (enable_saml_microservice | default(false))
     - role: arkcase-saml-authority
-      tags: [saml-authority]
-      when: enable_microservices | default(false)
+      tags: [arkcase-saml-authority]
+      when: enable_microservices | default(false) and enable_saml_microservice | default(false)
```

Comment:
This role (arkcase-saml-authority) should have the same when clause as arkcase-ldap-authority or as it is defined now?

Reply:
It's not used now at all. But it should be either SAML od LDAP authentication. We might need both at some point, but currently it's one or the other.